### DISCUSSION TITLE
Wallet address case sensitivity

### DISCUSSION THREAD

**Discussion by curiosthoth** [Web3Auth/Web3Auth#612] [Help]
Hi Web3Auth team,

I have a quick question regarding the address Web3Auth accepts for wallet challenges for Ethereum.

For a wallet address are you expecting a [checksum address ](https://github.com/ethereum/EIPs/issues/55#issuecomment-187159063)(https://ethsum.netlify.app/) ? Or is it case insensitive? Or prefer certain case?

E.g, 

`0xEdB14d75E5F06fEA3139e071817F615Eb6949d34`
vs
`0XEDB14D75E5F06FEA3139E071817F615EB6949D34`
vs
`0xedb14d75e5f06fea3139e071817f615eb6949d34`

Thank you

**Comment by himanshuchawla009** [comment]
Hey, you can pass in any case but make sure to pass the same case in verify api as well.
